### DISCUSSION TITLE
Control - getting pose from input image ?

### DISCUSSION THREAD

**Discussion by cgidesign-de** [vladmandic/sdnext#2955] [Q&A]
I am new to the build in Control.

I am trying to use an input image (normal image, not a pose) to extract the pose from it. This pose I like to use as input for creating new, final images.

How can this be done, if it is possible ?

**Comment by brknsoul** [comment]
Under the image viewers, expand "Control elements", then set them up this way;

![image](https://github.com/vladmandic/automatic/assets/1969381/cfb066d6-f1e1-47e9-8729-9f0155fa344f)


You can use the 🖼️ icon to generate a wireframe, in which you can save it for future use, or you can just generate the image normally.

**Comment by cgidesign-de** [comment]
Thanks, I'll try that.

**Comment by cgidesign-de** [comment]
I tried to set it up, but the ControlNet listbox is empty on my side. Do I need to download additional models for this?
![controlnet](https://github.com/vladmandic/automatic/assets/115958660/2c03c296-085f-47e2-bd2c-dafb03b75016)

**Reply by brknsoul** [reply]
Click the little refresh button to the right.

Just note, the Control Tab is sd.next's implementation of ControlNet, so you don't need the models, as they're downloaded by sd.next as needed.
